2020-07-13 16:18:55,001 ERROR 5396 [-/::1/-/128ms GET /car?username=%27muss%27] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'muss'%'' at line 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: select * from cartable where username like '%'muss'%'
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'muss'%'' at line 1"
sqlState: "42000"
index: 0
sql: "select * from cartable where username like '%'muss'%'"
name: "ER_PARSE_ERRORError"
pid: 5396
hostname: FH76NJO29HM8FWC

2020-07-13 22:52:14,894 ERROR 6468 [-/::1/-/10200ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-13 22:52:31,505 ERROR 6468 [-/::1/-/10006ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-13 22:53:55,483 ERROR 6468 [-/::1/-/10005ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-13 22:55:40,609 ERROR 6468 [-/::1/-/10005ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-13 23:03:46,022 ERROR 6468 [-/::1/-/10005ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-13 23:04:21,386 ERROR 6468 [-/::1/-/10005ms GET /allbooks] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as getConnection] (C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at Pool.query (C:\Users\wo\Desktop\project\dangdang\server\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\wo\Desktop\project\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    sql: select * from goods
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 6468
hostname: DESKTOP-6BKTIMC

2020-07-14 13:51:03,186 ERROR 6680 [-/::1/-/334ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'name' in 'field list'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set name = 'muss1' where uid=1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'name' in 'field list'"
sqlState: "42S22"
index: 0
sql: "update  user set name = 'muss1' where uid=1"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 6680
hostname: FH76NJO29HM8FWC

2020-07-14 13:52:02,372 ERROR 6680 [-/::1/-/5ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'name' in 'field list'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set name = 'muss1' where uid=1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'name' in 'field list'"
sqlState: "42S22"
index: 0
sql: "update  user set name = 'muss1' where uid=1"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 6680
hostname: FH76NJO29HM8FWC

2020-07-14 14:06:36,254 ERROR 5912 [-/::1/-/8ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'cccccc' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'cccccc' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:07:14,197 ERROR 5912 [-/::1/-/6ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'aaaaaaa' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'aaaaaaa' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:07:40,070 ERROR 5912 [-/::1/-/5ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'aaaaaaa' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'aaaaaaa' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:10:48,163 ERROR 5912 [-/::1/-/13ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'aaaaaaa' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'aaaaaaa' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:11:55,210 ERROR 5912 [-/::1/-/8ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'tiyiuffutfu' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'tiyiuffutfu' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:12:33,240 ERROR 5912 [-/::1/-/5ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'ssfsdfsadf' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'ssfsdfsadf' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:13:08,330 ERROR 5912 [-/::1/-/6ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'asdasdada' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'asdasdada' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 5912
hostname: FH76NJO29HM8FWC

2020-07-14 14:14:30,512 ERROR 7620 [-/::1/-/44ms POST /mychgkey] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'asdasdad' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'asdasdad' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 7620
hostname: FH76NJO29HM8FWC

2020-07-14 14:28:06,023 ERROR 7304 [-/::1/-/4ms POST /mychgusern] nodejs.TypeError: this.ctx.service.mychgkey.myChgUsers is not a function
    at MychgkeyController.myChgUser (C:\Users\Administrator\Desktop\app\dangdang\server\app\controller\mychgkey.js:11:49)
    at Object.callFn (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\koa-compose\index.js:45:18)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\koa-compose\index.js:44:32)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\koa-compose\index.js:36:12
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 7304
hostname: FH76NJO29HM8FWC

2020-07-14 14:42:01,518 ERROR 6752 [-/::1/-/6ms POST /mychgusern] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\app\dangdang\server\node_modules\co\index.js:54:5
    sql: update  user set username = 'qqqqqq' where uid=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "update  user set username = 'qqqqqq' where uid=undefined"
headers: {"Access-Control-Allow-Origin":"http://localhost:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 6752
hostname: FH76NJO29HM8FWC

